src/components/Launch/LaunchForm/SimpleInput.tsx

```diff
@@ -2,28 +2,43 @@ import {
     FormControl,
     FormControlLabel,
     FormHelperText,
+    MenuItem,
+    Select,
     Switch,
     TextField
 } from '@material-ui/core';
+import { makeStyles } from '@material-ui/core/styles';
 import * as React from 'react';
 import { DatetimeInput } from './DatetimeInput';
 import { makeStringChangeHandler, makeSwitchChangeHandler } from './handlers';
 import { InputProps, InputType } from './types';
 import { UnsupportedInput } from './UnsupportedInput';
 import { getLaunchInputId } from './utils';
 
+const useStyles = makeStyles(theme => ({
```

Comment:
NIT: not a blocker but do we need makeStyles here; couldn't we just send styles as a generic object?  (is there a specific reason for useStyle?)

eg, 
```
const formStyle = {
   minWidth: 100%; 
}
```

Reply:
If we define a style that way, we should use style props of React component.
But I found that we currently use className prop for styling.

That's why I went with such solution, just tried to replicate how it's already implemented.